package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "devDependencies": {
+    "vitepress": "^2.0.0-alpha.12"
```

Comment:
As far as I remember, we are not allowed to use alpha versions of the 3 party libraries.

Reply:
Why gardener was allowed to use it?